docs/create_allocator_service.md

```diff
@@ -0,0 +1,435 @@
+# Quickstart Create an Allocator Service
+
+This quickstart describes one way to implement a service which will allocate a Game Server on demand using the Agones API.  This allows a game client or matchmaker service to call the allocator service in order to allocate a Game Server and retrieve its IP address and port number. The game client can then connect directly to the Game Server, so that the minimized latency that is in the Agones design is preserved.
+
+## Objectives
+- Create a secure allocator service
+- Deploy the service to GKE
+- Allocate a Game Server from a Fleet using the service
+
+
+:warning: This quickstart requires Agones installed with agones.image.tag=0.5.0-03f4866 or newer :warning:
+
+## Prerequisites
+1. A [Go](https://golang.org/dl/) environment
+2. [Docker](https://www.docker.com/get-started/)
+3. Agones installed on GKE, with a simple-udp fleet
+4. kubectl properly configured
+5. A clone or fork of the [Agones repo](https://github.com/GoogleCloudPlatform/agones)
+
+
+>NOTE: Agones required Kubernetes versions 1.9 with role-based access controls (RBAC) and MutatingAdmissionWebhook features activated. To check your version, enter `kubectl version`.
+
+To install on GKE, follow the install instructions (if you haven't already) at
+[Setting up a Google Kubernetes Engine (GKE) cluster](../install/README.md#setting-up-a-google-kubernetes-engine-gke-cluster). Also complete the "Enabling creation of RBAC resources" and "Installing Agones" sets of instructions on the same page.
+
+While not required, you may wish to review the [Create a Game Server](../docs/create_gameserver.md), [Create Game Server Fleet](../docs/create_fleet.md), and/or [Edit a Game Server](../docs/edit_first_game_server.md) quickstarts.
+
+
+### 1. Create Firewall Rules
```

Comment:
I was reading https://cloud.google.com/load-balancing/docs/health-checks which has this bit...
```
HTTP(S), SSL Proxy, TCP Proxy, and Internal Load Balancing

When a health check is used with HTTP(S), SSL Proxy, TCP Proxy, or Internal Load Balancing, the health check probes come from addresses in the ranges 130.211.0.0/22 and 35.191.0.0/16. You need to create firewall rules that allow these connections to all your load balanced instances.
```
so I thought I'd better add it since we are using https.  I haven't tried running it without the firewall.  Is this not applicable to this service?